csharp/tools/tracing-config.lua

```diff
@@ -48,6 +49,14 @@ function RegisterExtractorPack(id)
                     dotnetRunNeedsSeparator = true
                     dotnetRunInjectionIndex = i + 1
                 end
+                if arg == 'test' then
+                    match = true
+                end
+                -- for `dotnet test`, we should not append `-p:UseSharedCompilation=false` to the command line
+                -- if a library or executable is being provided as an argument.
+                if arg:match('%.exe$') or arg:match('%.dll')  then
+                    libOrExe = true
+                end
```

Comment:
I guess this means that it will apply to `dotnet run` as well, which I guess is good?

Reply:
It would work, but it is possible to also provide the `-p:UseSharedCompilation=false` flag to `dotnet run` when it targets a `dll` or `exe` file. I will change the code such that we only skip the injection of the flag for `dotnet test`.